schemas/aws/vpc-request-1.yml

```diff
@@ -45,6 +45,16 @@ properties:
         type: array
         items:
           type: string
+      private_subnet_tags:
+        description: Private subnets tags.
+        type: object
+      public_subnet_tags:
+        description: Public subnets tags.
+        type: object
```

Comment:
do you want to keep these 2 in root level to be consistent with `schemas/aws/vpc-1.yml` file, or move vpc-1.yml ones to be under `subnets`

Reply:
is there a use case that each subnet to have different tags? we can even add `tags` to subnet object

Reply:
Yes, 

These tags are used to auto-discover subnets, and there are cases where subnets (or AZs) need to be excluded. For example, RDS Proxy is not supported in us-east-1e 

[Ref](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/rds-proxy.html#rds-proxy.RegionVersionAvailability): 
```
Certain Regions have Availability-Zone (AZ) restrictions to consider while creating your proxy. US East (N. Virginia) Region does not support RDS Proxy in the use1-az3 Availability Zone. US West (N. California) Region does not support RDS Proxy in the usw1-az2 Availability Zone. When selecting subnets while creating your proxy, make sure that you don't select subnets in the Availability Zones mentioned above.
```

The tf module also allows setting tags per az with `private_subnet_tags_per_az`